docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,59 @@ tags:
 
 :::tip Support, stability, and dependency info
 
-- In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API Keys used for Cloud Operations Tools (tcld, API, Terraform) are In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
+- API Keys used for Namespace Authentication are In [Pre Release](/evaluate/development-production-features/release-stages#pre-release)
 
 :::
 
 Temporal Cloud supports secure programmatic access through API Key Authentication.
 
 ## Overview
 
-Temporal Cloud API Keys facilitate user-level authentication.
-Each key is unique to a single user and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
+Temporal Cloud API Keys provide identity, User and [Service Account](/cloud/service-accounts), based authentication.
```

Comment:
what does it mean to say identity, User and Service Account authentication?

Reply:
IIUC, a User and a Service Account are Identity Types.  
We use Identities in the product at https://cloud.temporal.io/settings/identities 
The Identity language in the docs aligns to the language in the product.

Reply:
I think there is a colon missing here because the sentence as constructed suggests there are 3 distinct items where as there is really Identity: User and Service Account

Reply:
ack - will make this more clear

Reply:
language changed. marking this one done